pipeline/builder/builder_test.go

```diff
@@ -201,6 +201,46 @@ func TestBuilderPipelineStages(t *testing.T) {
 
 			expected := map[string]string{"hello": "world", "test": "annotations"}
 			assert.Equal(t, expected, spinnaker.Stages[0].(*types.DeployStage).Clusters[0].PodAnnotations)
+			assert.Equal(t, (*types.Deployment)(nil), spinnaker.Stages[0].(*types.DeployStage).Clusters[0].Deployment)
+		})
+
+		t.Run("Deployments are functional", func(t *testing.T) {
```

Comment:
"are functional" is an anti pattern for test names. I'd say "Deployments are appended to the struct when included in config"